project_task_autocreate/models/project_task_template.py

```diff
@@ -286,7 +286,7 @@ def get_task_vals(self, record=None):
         """
         description = self.description
         if description:
-            description = unicode(description).format(object=record)
+            description = str(description).format(object=record)
```

Comment:
@rimbendhaou The `unicode()` was added to solve a encoding issue when using accents. If you remove it it will cause a regression.